### PR TITLE
Style core elements

### PR DESCRIPTION
**Pull request by carlos-osorio-developer**
## Add functionality for:

- TailwindCSS framework.
- Styles for index.html.

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -1,3 +1,662 @@
...
+
+/**
+Remove the additional ':invalid' styles in Firefox.
+See: https://github.com/mozilla/gecko-dev/blob/2f9eacd9d3d995c937b4251a5557d95d494c9be1/layout/style/res/forms.css#L728-L737
+*/
+
+/**
+Remove the padding so developers are not caught out when they zero out 'fieldset' elements in all browsers.
+*/
+
```
Comment (henatan99) [inline assets/css/main.css]: - [x] There are too many commented lines. Although there are very informative it is not the best practice way to commit too much or necessary commented code line. You can branch out and keep the informative comments for yourself and make your file cleaner.

File: assets/css/main.css [hunk 2]
```diff
@@ -1,3 +1,662 @@
...
+h3,
+h4,
+h5,
+h6,
+hr,
+figure,
+p,
+pre {
+  margin: 0;
+}
```
Comment (henatan99) [inline assets/css/main.css]: - [x] OPTIONAL: I agree general styling of element selectors does a good job in styling. But, I think it is better you use class selectors for much of the styling and do the general styling when it is strictly required.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @carlos-osorio-developer , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
You did a very good job. Just make the minor changes and you are good to go. 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by umararshadbut — APPROVED** [review]
Hi @carlos-osorio-developer and @pountzas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
HAPPY CODING :computer:
**Umair Arshad**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**